### ISSUE TITLE
Loop status-var not compiling everytime

### ISSUE THREAD

**Issue by RSWilli** [marko-js/marko#904]
### Marko Version: 4.5.3
Serverside Rendering

### Details
The loop status variable is not compiling when the equal sign is padded in spacing

### Expected Behavior
`<li for(result in input.results | status-var = loop)>itemtext</li>`
to do the same as
`<li for(result in input.results | status-var=loop)>itemtext</li>`
(Mind the spacing for status-var)

### Actual Behavior
`<li for(result in input.results | status-var = loop)>itemtext</li>`
Marko throws:

> Error: An error occurred while trying to compile template at path "PATH_TO_TEMPLATE.marko". Error(s) in template: 1) [template.marko] Unexpected input: status-var = loop

Personally I prefer the spaced variant because it improves readability. 

I think it would be more user friendly if both versions were possible

**Comment by patrick-steele-idem** [comment]
Thanks for the bug report. That does seem like a problem. On a related note, you can do the following:

```marko
<li for(index,result in input.results)>
    ${index} - ${result}
</li>
```

**Comment by patrick-steele-idem** [comment]
Relevant source code: https://github.com/marko-js/marko/blob/master/src/taglibs/core/util/parseFor.js
Relevant tests: https://github.com/marko-js/marko/blob/master/test/parseFor-test.js

**Comment by austinkelleher** [comment]
Fixed in `marko@marko@4.5.4`.
